test/lib/sstable_utils.hh

```diff
@@ -163,6 +163,11 @@ public:
         _sst->_shards.push_back(this_shard_id());
     }
 
+    void set_first_last(const dht::decorated_key& first_key, const dht::decorated_key& last_key) {
```

Comment:
The name is a bit terse.
How about `set_first_and_last_keys`.
Can be used even by today's sstable::set_first_and_last_keys() and be exposed to tests.